extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -24,14 +24,35 @@ import {closest, removeElement} from '../../../src/dom';
 import {dev, devAssert} from '../../../src/log';
 import {getLocalizationService} from './amp-story-localization-service';
 import {getState} from '../../../src/history';
-import {htmlFor} from '../../../src/static-template';
+import {htmlFor, htmlRefs} from '../../../src/static-template';
+import {buildOpenAttachmentElementLinkIcon} from './amp-story-open-page-attachment';
+import {setImportantStyles} from '../../../src/style';
 import {isPageAttachmentUiV2ExperimentOn} from './amp-story-page-attachment-ui-v2';
 import {toggle} from '../../../src/style';
 import {triggerClickFromLightDom} from './utils';
 
 /** @const {string} */
 const DARK_THEME_CLASS = 'i-amphtml-story-draggable-drawer-theme-dark';
 
+/**
+ * Distance in pixels to swipe before opening attachment.
+ * @const {number}
+ */
+const OPEN_THRESHOLD = 150;
```

Comment:
nit, here and below:

```suggestion
const OPEN_THRESHOLD_PX = 150;
```